src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+
+export interface SearchFormProps {
+    LabelContent: string
+    spanContent: string
+}
+
+const SearchForm: React.FC<SearchFormProps> = ({ LabelContent, spanContent }): JSX.Element => {
+    return (
+        <form className="c-searchform" role="search" method="get" action="https://carleton.ca">
+            <label htmlFor="searchform__input">{LabelContent}</label>
+            <input id="searchform__input" name="s" type="search" autoComplete="off" placeholder="Search" />
```

Comment:
This should have the placeholder props value.
Please create one.

Reply:
Done

Reply:
Please create an optional placeholder props.

Reply:
Done